src/pages/edge/adobe-journey-optimizer/content-card-ui/iOS/templates/smallimage-template.md

```diff
@@ -0,0 +1,48 @@
+---
+title: SmallImageTemplate
+description: This class represents a SmallImage templated content card authored in Adobe Journey Optimizer.
+keywords:
+- Adobe Journey Optimizer
+- Guide
+- Content Card
+- Messaging
+- Card Templates
+- Content Card Templates
+- Small Image Template
+- AEPStack
+- AEPHStack
+- AEPVStack
+- AEPText
+- AEPButton
+- AEPImage
+- ButtonHStack
+- RootHStack
+- TextVStack
+---
+
+# SmallImageTemplate
+ 
+ This class represents a SmallImage templated content card authored in Adobe Journey Optimizer. 
+ 
+ A small image template content card includes a title, body, image, and a maximum of three buttons. The image is displayed in line with the text content.  An optional dismiss button can be added to dismiss the content card. 
+ 
+ Use the SmallImageTemplate class to customize the appearance of the small image templated content cards. 
+ 
+ SmallImageTemplate conforms to `ObservableObject`, allowing it to be used reactively in SwiftUI views.
+
+## Layout
+
+<img src="../../assets/iOS/smallimagetemplate-layout.png" width="500"/>
+
+## Public properties
+
+| Property      | Type                                           | Description                                                  |
+| ------------- | ---------------------------------------------- | ------------------------------------------------------------ |
+| title         | [AEPText](../ui-elements/aeptext.md)            | The title text for the content card.                         |
+| body          | [AEPText](../ui-elements/aeptext.md)            | *Optional*<br/>The body text of the content card                 |
+| image         | [AEPImage](../ui-elements/aepimage.md)          | *Optional*<br/>The image to be shown on the content card.     |
+| buttons       | [[AEPButton](../ui-elements/aepbutton.md)]      | *Optional*<br/>The list of buttons on the content card.       |
```

Comment:
<img width="306" alt="Screenshot 2024-11-07 at 11 13 57 PM" src="https://github.com/user-attachments/assets/622efd97-7004-4abe-9a42-780c18746cc3">

<img width="316" alt="Screenshot 2024-11-07 at 11 14 51 PM" src="https://github.com/user-attachments/assets/f794f426-da74-402b-a727-b904d2df2aba">


To me a new line makes it look a little cleaner. Let me know otherwise or some other way to make it better

Reply:
A nice option here to make this cleaner would be to add a fourth column in the table titled "Required", providing "yes" and "no" respectively where the properties are or are not required.